codecov.yml

```diff
@@ -2,11 +2,10 @@ ignore:
   - "**/__init__.py"
   - "pylibczi/tests"
   - "pylibczi/bin"
-  - "pylibczi/CziScene.py"
   - "pylibczi/exceptions.py"
   - "pylibczi/types.py"
```

Comment:
I know these weren't touched as a part of this PR but I feel like these should be not ignored by codecov personally.